.vsts-ci/templates/pushPhase.yml

```
@@ -33,52 +76,54 @@ jobs:
       }
     displayName: Verify that only one snap was downloaded
 
-  - powershell: |
+  - pwsh: |
       [string]$Branch=$env:BUILD_SOURCEBRANCH
       $branchOnly = $Branch -replace '^refs/heads/';
       $branchOnly = $branchOnly -replace '[_\-]'
 
-      if($branchOnly -eq 'master' -or $branchOnly -like '*dailytest*')
-      {
-          Write-verbose "release branch:" -verbose
-          $generatedBranch = (get-date).ToString('yyyyMMddHH')
-          $releaseTag = "edge/$generatedBranch"
-          $vstsCommandString = "vso[task.setvariable variable=ReleaseTag]$releaseTag"
-          Write-Verbose -Message "setting ReleaseTag to $releaseTag" -Verbose
-          Write-Host -Object "##$vstsCommandString"
-      }
-      else
-      {
-          Write-verbose "non-release branch" -verbose
-          # Branch is named <previewname>
-          $releaseTag = "edge/$branchOnly"
-          $vstsCommandString = "vso[task.setvariable variable=ReleaseTag]$releaseTag"
-          Write-Verbose -Message "setting ReleaseTag to $releaseTag" -Verbose
-          Write-Host -Object "##$vstsCommandString"
+      if('${{ parameters.release }}' -eq 'private') {
+        if($branchOnly -eq 'master' -or $branchOnly -like '*dailytest*')
+        {
+            Write-verbose "release branch:" -verbose
```

do you mean to have a value for release branch specified in the verbose message here? perhaps:
```suggestion
            Write-verbose "release branch:" $branchOnly -verbose
```